src/ray/gcs/tables.h

```diff
@@ -250,13 +313,14 @@ Status TaskTableTestAndUpdate(AsyncGcsClient *gcs_client, const TaskID &task_id,
                               SchedulingState update_state,
                               const TaskTable::TestAndUpdateCallback &callback);
 
-class ClientTable : private Table<ClientID, ClientTableData> {
+class ClientTable : private Log<UniqueID, ClientTableData> {
```

Comment:
Should this be `ClientID`?  I didn't check deeper.  But for readers like me who aren't intimately familiar with this code, a `class ClientTable` seems to suggest `ClientID` being keys more.

Reply:
I changed it to `UniqueID` because the log of client table entries is stored at a single key, which should not be thought of as a `ClientID`. I can add some documentation for that.